<?xml version='1.0'?>
<launch>
    <arg name ="ns" default ="agent1"/>
  	<arg name="agent_costmap_topic" default="map"/>
  	<arg name="agent_costmap_updates_topic" default="move_base/global_costmap/map_updates"/>

    <group ns="$(arg ns)">
        <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
            <param name="robot_base_frame" value="$(arg ns)/base_link"/>
            <param name="costmap_topic" value="$(arg agent_costmap_topic)"/>
            <param name="costmap_updates_topic" value="$(arg agent_costmap_updates_topic)"/>
            <param name="visualize" value="true"/>
            <param name="planner_frequency" value="0.33"/>
            <param name="progress_timeout" value="30.0"/>
            <param name="potential_scale" value="3.0"/>
            <param name="orientation_scale" value="0.0"/>
            <param name="gain_scale" value="1.0"/>
            <param name="transform_tolerance" value="0.3"/>
            <param name="min_frontier_size" value="0.40"/>
        </node>
    </group>

</launch>